backend/frontend/forms.py

```diff
@@ -0,0 +1,48 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.signing import TimestampSigner
+from django.urls import reverse_lazy
+from django.contrib.auth.forms import SetPasswordForm, AuthenticationForm
+
+from .theme import styles
+
+
+User = get_user_model()
+
+
+class ThemedFormMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        for field in self.fields:
+            self.fields[field].widget.attrs.update(
+                {
+                    "class": styles["input_small"],
+                }
+            )
```

Comment:
This is a workaround for adding certain styles/classes on the fly?

Reply:
yes, so that leaving django to render the form, we'd still get out custom classes